content/release-notes/2.11.2.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Haystack 2.11.2
+description: Release notes for Haystack 2.11.2
+featured_image: /images/release-notes.png
+images: ["/images/release-notes.png"]
+toc: True
+date: 2025-03-18
+last_updated: 2025-03-18
+tags: ["Release Notes"]
+link: https://github.com/deepset-ai/haystack/releases/tag/v2.11.2
+---
+
+# Release Notes
+
+## v2.11.2
+
+### Enhancement Notes
+
+-  Refactored the processing of streaming chunks from OpenAI to simplify logic.
+-  Added tests to ensure expected behavior when handling streaming chunks when using include_usage=True.
+
+### Bug Fixes
+
+-   Fixed issue with MistralChatGenerator not returning a <span class="title-ref">finish_reason</span> when using streaming. Fixed by adjusting how we look for the <span class="title-ref">finish_reason</span> when processing streaming chunks. Now, the last non-None <span class="title-ref">finish_reason</span> is used to handle differences between OpenAI and Mistral.
```

Comment:
could you please stick to this format?

https://github.com/deepset-ai/haystack-home/blob/main/content/release-notes/2.11.0.md

Reply:
will do next time!